docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -2,7 +2,8 @@
 aliases:
   - meta-monitoring/
```

Comment:
I'm not sure this alias is doing anything. I am pretty sure it would have been redirecting https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/ to itself.

I think it can be removed unless it was actually meant to redirect something like https://grafana.com/docs/grafana/latest/alerting/meta-monitoring/ to this page?

Reply:
done!